src/cljs/reagent_toolbox/core.cljs

```diff
@@ -55,19 +55,39 @@
 
 (def card-title (reagent/adapt-react-class (.-CardTitle js/ReactToolbox)))
 
-(def checkbox (reagent/adapt-react-class (.-Checkbox js/ReactToolbox)))
+(def checkbox-component (reagent/adapt-react-class (.-Checkbox js/ReactToolbox)))
+
+(defn checkbox
+  [properties]
```

Comment:
If there are no docs and no variable arity, I prefer to put the function definitions in one line. I tend to like compact code. This is not a big deal.